web-components/src/components/header/NavLink.tsx

```diff
@@ -26,16 +26,18 @@ export const useNavLinkStyles = makeStyles<Theme, { isActive: boolean }>(
 
 export type NavLinkProps = {
   href: string;
+  pathname: string;
```

Comment:
I'm not sure why this hasn't happened before, but something about having the `nct` nav link in the highest level (not handled through one of the dropdown helper components like the others) messed up gatsby's build - there's no `window` object at build time, so the `window.location.pathname` check errored.

This is a workaround, but i'm also noticing i'm not very happy with the helper functions I built for nav - initially they were a sort of hack to work the new design into an older one, but the entire nav setup seems ripe for refactoring